### PR TITLE
Deploying travis

### PR DESCRIPTION
**Pull request by Anies12**
removed a space before "script" , travis was unable to parse the .yml file

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,25 +1,18 @@
...
  - npm i
  - psql -c "create database travis_ci_test;"
  - psql -c "create user travis with password '123456';" -U
-
 notifications:
   email: false
-
 env:
   global:
   - PGPORT=5432
```
Comment (kannanhassouna) [inline .travis.yml]: white space

**Review by kannanhassouna — CHANGES_REQUESTED** [review]
you have a problem with Travis  , please review #52

**Review by Fatmasiam — APPROVED** [review]

**Review by Angham116 — APPROVED** [review]

**Review by ahmedalmi95 — APPROVED** [review]

File: test/test.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+const test = require('tape');
+test('asd',(t)=> {
+    t.equal(1,1,'dfgd');
+    t.end();
+})
```
Comment (kannanhassouna) [inline test/test.js:5]: insert white space

File: test/test.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+const test = require('tape');
+test('asd',(t)=> {
```
Comment (kannanhassouna) [inline test/test.js:2]: use Eslint

File: package.json [hunk 4]
```diff
@@ -31,6 +31,8 @@
     "eslint": "^5.15.3",
     "eslint-config-airbnb-base": "^13.1.0",
     "eslint-plugin-import": "^2.16.0",
-    "nodemon": "^1.18.10"
+    "nodemon": "^1.18.10",
```
Comment (kannanhassouna) [inline package.json:34]: remove  ``` nodemon ``` 
your pull request about travis and testing why you add ``` nodemon ```

**Review by kannanhassouna — CHANGES_REQUESTED** [review]

**Review by Fatmasiam — CHANGES_REQUESTED** [review]
there is conflict try to solve it

**Pull request closed without merging**